microprofile/tests/tck/tck-config/pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>helidon-microprofile-config</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why is this here for config TCK? AFAIK metrics are not used from config

Reply:
As long as you have this dependency, you need metrics:
```
        <dependency>
            <groupId>io.helidon.microprofile.tests</groupId>
            <artifactId>helidon-arquillian</artifactId>
        </dependency>

```
We could add it directly in Helidon-Arquillian but I wanted to remove any microprofile dependency from there.